src/OrchardCore.Modules/OrchardCore.Workflows/Expressions/LiquidWorkflowExpressionEvaluator.cs

```diff
@@ -33,14 +34,17 @@ ILogger<LiquidWorkflowExpressionEvaluator> logger
 
         private IStringLocalizer T { get; }
 
-        public async Task<T> EvaluateAsync<T>(WorkflowExpression<T> expression, WorkflowExecutionContext workflowContext)
+        public async Task<T> EvaluateAsync<T>(WorkflowExpression<T> expression, WorkflowExecutionContext workflowContext, TextEncoder encoder = null)
         {
+            if(encoder == null)
+                encoder = System.Text.Encodings.Web.JavaScriptEncoder.Default;
```

Comment:
We always use braces for an if block but i agree not so important ;)

Reply:
Done ;)